psi4/driver/procrouting/scf_proc/subclass_methods.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+from psi4 import core
+
+def _RHF_orbital_gradient(self, save_fock: bool, max_diis_vectors: int) -> float:
+    gradient = self.form_FDSmSDF(self.Fa(), self.Da())
+
+    if save_fock:
+        if not self.initialized_diis_manager_:
+            storage_policy = getattr(core.DIISManager.StoragePolicy, "InCore" if self.scf_type() == "DIRECT" else "OnDisk")
```

Comment:
```suggestion
            if self.scf_type() == "DIRECT":
                storage_policy = core.DIISManager.StoragePolicy.InCore 
            else:
                storage_policy = core.DIISManager.StoragePolicy.OnDisk
```
equivalent? I usually save `getattr` for when attribute only known in a variable. But that's my readability opinion. Your choice.

Reply:
Yes, equivalent. The right thing to do for readability is probably to import `from core.DIISManager import StoragePolicy, RemovalPolicy`. Then I can have a terse ternary without using `getattr`.

Reply:
that works. only other consideration is if you'd ever want to track which paths were tested with codecov.